### DISCUSSION TITLE
Specify changes in partials in the release description

### DISCUSSION THREAD

**Discussion by wilhelmer** [squidfunk/mkdocs-material#4428] [Ideas]
Posting here since it's not really an issue or feature suggestion:

In the release description, could you please specify which partials have been changed, if any?

Maybe also indicate whether the changes are minor, major, or breaking.

This would be very useful for users who are overriding partials and want to keep them in sync with the latest version.

Thanks!

**Comment by squidfunk** [comment]
Good idea, I'll try to integrate that into the process. Releases are manual, so I'll add it to my checklist. If I forget, please don't hesitate to remind me again 😅 Or maybe just check out the compare of the new version with the last? e.g.: https://github.com/squidfunk/mkdocs-material/compare/8.5.2...8.5.3

**Comment by wilhelmer** [comment]
Yes, I can check that via GitHub, but that's pretty cumbersome, plus I don't know whether it's a breaking change or just a small bugfix. It would be great to know whether it's safe to update just by reading the release notes.

**Reply by squidfunk** [reply]
It's very hard for me to decide whether it is a breaking change or not, as I don't know what the author did to the partials. In theory, everything changed on a partial could be a breaking change. Additionally, if the partials were adjusted, every code line changed in stylesheets or JavaScript might be a breaking change. This is exactly what makes this project so tedious to maintain.

If we'd classify those as such, we'd already be at major version 256 or so, so I don't consider changes to partials breaking changes. Let me think about this and see whether I can come up with something that is of value, but currently, I don't think listing the partials would help much. Maybe it's better to explain the upgrade process of minor versions in the documentation and link to the compare view. Furthermore, the compare view offers the possibility to compare two specific versions, not only from `n-1` to `n`.

**Reply by wilhelmer** [reply]
Forget about the minor/major/breaking thing then, simply listing the partials is already super helpful and solves 95% of the problem. I think most users will only override 1-2 partials, so most of the time, they will be able to glance at the release notes, see that their partial hasn't changed, and safely update to that version.

Use something like `git diff --name-only 8.5.2..HEAD src/partials` and copy-paste the output to the release notes 😉

**Comment by wilhelmer** [comment]
(I know you probably won't be able to tell exactly whether it will break my project or not, maybe just a rough indicator like "minor"/"major". Or leave that aside and just list the partials, that will already help.)
